polyfills/Array/from/tests.js

```diff
@@ -49,42 +49,49 @@ describe('returns an array with', function () {
 		var map;
 		var mapIterator;
 
-		if ('Map' in window && 'entries' in Map.prototype) {
+		function returnArgs(){
+			return Array.prototype.slice.call(arguments);
+		}
 
+		if ('Map' in window) {
 			map = new Map();
 			map.set(1,2);
 			map.set(3,4);
-			mapIterator = map.values();
-
-			if (typeof mapIterator.next === 'function') {
+			if (map.values) {
+				mapIterator = map.values();
 
-				// Test map iterable
-				proclaim.deepEqual(Array.from(mapIterator), [2,4]);
+				if (typeof mapIterator.next === 'function') {
 
-				it('can convert from Map', function () {
-					proclaim.deepEqual(Array.from(map), [[1,2],[3,4]]);
-				});
+					// Test map iterable
+					proclaim.deepEqual(Array.from(mapIterator), [2,4]);
```

Comment:
How come this test is now using the map iterator instead of the map itself? Could we have a comment which explains the reasoning please